action/protocol/execution/evm/evm.go

```diff
@@ -170,13 +183,19 @@ func newParams(
 			GasPrice: execution.GasPrice(),
 		},
 		execution.Nonce(),
-		protocol.MustGetBlockchainCtx(ctx).EvmNetworkID,
+		evmNetworkID,
```

Comment:
since `protocol.MustGetBlockchainCtx(ctx)` is only called/used 1 time in here, no need to make this change